src/main/kotlin/io/github/mojira/arisa/ModuleExecutor.kt

```diff
@@ -50,29 +52,30 @@ class ModuleExecutor(
     private val config: Config,
     private val cache: Cache
 ) {
-    private val attachmentModule: AttachmentModule =
+    private val attachmentModule =
```

Comment:
Actually seems like all autoformat rules are a bit wrong :/

Reply:
Did you leave those intentionally?
I removed them on purpose with IntelliJ, since I see no value in `val attachmentModule: AttachmentModule = AttchamentModule(...`